src/Npgsql/NpgsqlActivitySource.cs

```diff
@@ -122,25 +126,31 @@ internal static void ReceivedFirstResponse(Activity activity, NpgsqlTracingOptio
         activity.AddEvent(activityEvent);
     }
 
-    internal static void CommandStop(Activity activity)
-    {
-        activity.SetStatus(ActivityStatusCode.Ok);
-        activity.Dispose();
-    }
-
-    internal static void SetException(Activity activity, Exception ex, bool escaped = true)
+    internal static void SetException(Activity activity, Exception exception, bool escaped = true)
     {
-        // TODO: We can instead use Activity.AddException whenever we start using .NET 9
-        var tags = new ActivityTagsCollection
+        var tags = new TagList
         {
-            { "exception.type", ex.GetType().FullName },
-            { "exception.message", ex.Message },
-            { "exception.stacktrace", ex.ToString() },
```

Comment:
@vonzshik any recollection what `exception.escaped` is for? I think we should probably remove it, especially since we're now passing through the .NET 9 standard AddException(), unlike before (another advantage of DiagnosticSource 9.0).